.github/workflows/distribution.yml

```diff
@@ -53,6 +53,8 @@ jobs:
         path: dist/
     - name: Build source distribution
       run: |
+        python -c 'import setuptools ; print("setuptools version", setuptools.__version__)'
```

Comment:
Given the annoyance here, I think we can leave this print statement in the final PR to aid our future selves.